tests/test_sorting.py

```diff
@@ -0,0 +1,16 @@
+from . import v
+assert v  # Silence flake 8
+
+
+def test_sorting(v):
+    v.scan("""\
+def hello():
+    pass
+
+def world():
+    print("Hello, I am a long function.")
+    return "World"
+""")
+    output = [item.name for item in v.get_unused_code(sort_by_size=True)]
+    expected = ['hello', 'world']
+    assert output == expected
```

Comment:
Let's make hello() longer than world(). Then we can have two tests: `assert [item.name for item in v.get_unused_code(sort_by_size=False)] == ['hello', 'world']` and `assert [item.name for item in v.get_unused_code(sort_by_size=True)] == ['world', 'hello']`. Also, let's use 'foo' and 'bar' instead of 'hello' and 'world'. "Hello world" is generally used for simple programs, not for testing.

Reply:
Agreed!